CsCore/CsCoreUnity/Plugins/package.json

```diff
@@ -2,11 +2,16 @@
 	"name": "com.csutil.cscore.unity",
 	"version": "1.8.8",
 	"displayName": "cscore Unity",
-	"description": "cscore is a minimal-footprint library providing commonly used helpers & patterns for your C# projects.",	
+	"description": "cscore is a minimal-footprint library providing commonly used helpers & patterns for your C# projects.",
 	"unity": "2018.1",
 	"license": "Apache License 2.0",
 	"author": {
 		"name": "csutil.com",
 		"url": "https://github.com/cs-util-com/cscore"
+	},
+	"dependencies": {
+		"com.csutil.cscore": "1.8.8",
+		"com.unity.modules.screencapture": "1.0.0",
+		"com.unity.modules.physics": "1.0.0"
```

Comment:
Are these new ones here really needed?

Reply:
The program does not compile as soon as you remove those packages via the paackage manager. Screencapture is needed by one obscure line in the logging helper you wrote.

Reply:
![grafik](https://user-images.githubusercontent.com/16704657/223364386-c3981f23-8abb-4f2a-b0d2-1249d61d8e67.png)
Screencapture

Reply:
![grafik](https://user-images.githubusercontent.com/16704657/223364827-20c6e436-2d02-44a0-8d68-396ec02e8c7f.png)
Physics

Reply:
I just dug into it and there seems to be even more:

![grafik](https://user-images.githubusercontent.com/16704657/223365758-fd9975cc-0a51-493b-83fd-4e3698eb9d8a.png)
Physics2D

![grafik](https://user-images.githubusercontent.com/16704657/223365998-709967da-ed0a-4d44-88b5-0ae6c641eb0d.png)
WebRequestTexture

Reply:
Ok interesting, maybe since these are tpyically added to all projects by default this wasnt noticed yet that they should be marked as dependencies

Reply:
Could be, yes. But WebGL builds are notoriously slow (a literally empty project takes around 10-15minutes to build) so we stripped everything we could to get the build times managable